docs/sphinx/conf.py.in

```diff
@@ -128,15 +113,14 @@ def popen(args, stdin=None, stdout=subprocess.PIPE, stderr=subprocess.PIPE):
 #modindex_common_prefix = []
 
 # Variables used to define Github extlinks
-if "SOURCE_BRANCH" in os.environ and len(os.environ.get('SOURCE_BRANCH')) > 0:
-    source_branch = os.environ.get('SOURCE_BRANCH')
-else:
-    source_branch = 'develop'
-
-if "SOURCE_USER" in os.environ and len(os.environ.get('SOURCE_USER')) > 0:
-    user = os.environ.get('SOURCE_USER')
-else:
-    user = 'openmicroscopy'
+source_branch = 'develop'
+if (ext_source_branch is not None and len(ext_source_branch) > 0 and
+    ext_source_branch != '${sphinx_source_branch}'):
```

Comment:
All understood. But at the moment, this token is also replaced by the `replace` task in the ant target so the check becomes something like:

```
-bash-4.1$ grep 5.1.0 conf.py
ext_source_branch = 'v5.1.0'
    ext_source_branch != 'v5.1.0'):
```